docs/book/v3/features/error-handling.md

```diff
@@ -51,6 +51,8 @@ application:
 $app->pipe($errorMiddleware);
 ```
 
+## There is an easier way
```

Comment:
Excerpt from the Writers Guide:

> For headlines the Chicago Manual of Style capitalization rules must be used:
> 
> 1. Capitalize the first and the last word.
> 1. Capitalize nouns, pronouns, adjectives, verbs, adverbs, and subordinate conjunctions.
> 1. Lowercase articles (a, an, the), coordinating conjunctions, and prepositions.
> 1. Lowercase the ‘to’ in an infinitive (I want to play guitar).
> 
> The online tool [capitalizemytitle.com](https://capitalizemytitle.com/style/Chicago/) can help to capitalize and case convert the headlines.
> 

```suggestion
## There Is an Easier Way
```

Some other suggestions for this section:

```suggestion
## Ready-Made Solution
```

```suggestion
## Use of the Ready-Made Handler
```

```suggestion
## Using the Prepared Handler
```

Reply:
Thanks for the feedback (referring to all of it here). When did the Chicago Manual of Style become the default? Apologies if I suggested it and forgot. :-P